pkg/policy/l4.go

```diff
@@ -59,52 +142,18 @@ func CreateL4Filter(rule api.PortRule, port api.PortProtocol, direction string,
 	}
 
 	if rule.Rules != nil {
-		l7rules := []AuxRule{}
-		for _, h := range rule.Rules.HTTP {
-			r := AuxRule{}
-
-			if h.Path != "" {
-				r.Expr = "PathRegexp(\"" + h.Path + "\")"
-			}
-
-			if h.Method != "" {
-				if r.Expr != "" {
-					r.Expr += " && "
-				}
-				r.Expr += "MethodRegexp(\"" + h.Method + "\")"
+		if rule.Rules.HTTP != nil {
```

Comment:
Such a structure:
```
if <condition 1> {
...
} else <condition 2> {
...
}
```
Can be rewritten a bit more elegantly into:
```
switch {
case <condition 1>:
...
case <condition 2>:
...
}
```

Reply:
done